.github/workflows/pypi_publish.yml

```
@@ -59,3 +129,209 @@
+    # Create a PR against the Builder, to update the CDK version that it uses.
+    # In the future, Builder may use the SDM docker image instead of the Python CDK package.
+    needs:
+      - build
+      - publish
+    if: (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')) || (github.event_name == 'workflow_dispatch' && github.event.inputs.update_connector_builder == 'true')
+    env:
+      VERSION: ${{ needs.build.outputs.VERSION }}
+      IS_PRERELEASE: ${{ needs.build.outputs.IS_PRERELEASE }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)